pkg/apk/apk/index.go

```diff
@@ -32,6 +32,7 @@ import (
 	"time"
 
 	"github.com/chainguard-dev/clog"
+	"github.com/charmbracelet/log"
```

Comment:
Need to `clog.FromContext()` to avoid pulling this in.

Reply:
Done